sdk/eventgrid/eventgrid/src/logger.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createClientLogger } from "@azure/logger";
+
+/**
+ * The @azure/logger configuration for this package.
+ */
+export const logger = createClientLogger("messaging-eventgrid");
```

Comment:
interesting that this prefix is different than the package name

Reply:
Wasn't sure what the convention here was, happy to just use `eventgrid` if our normal strategy is to just use the package name.

Reply:
I think that's the most sane and prevents two packages from getting confused about what namespace to log to